### PR TITLE
Store "audience" and "attributedTo" data

### PR DESCRIPTION
**Pull request by annando**
While analyzing the Lemmy issue with the missing pictures, I came across the "audience" field that we haven't handled before. We now store it and will see if we possibly use it at a later point in time. Also for debugging purposed we now store the "attributedTo" in a different place, since according to the specs this can be an array - although most implementations don't use an array for that. Possibly we will use the content in the field at a later point in time.

Since this isn't a bugfix, we possibly should postpone this PR to the next version.

### CODE REVIEW

File: src/Module/PermissionTooltip.php [hunk 1]
```diff
@@ -236,7 +236,13 @@ private function fetchReceivers(int $uriId): string
...
 					$output .= DI::l10n()->t('<b>BCC:</b> %s<br>', implode(', ', $receiver));
 					break;
-			}
+				case Tag::AUDIENCE:
+					$output .= DI::l10n()->t('<b>Audience:</b> %s<br>', implode(', ', $receiver));
+					break;
+				case Tag::ATTRIBUTED:
+					$output .= DI::l10n()->t('<b>Attributed To:</b> %s<br>', implode(', ', $receiver));
+					break;
+				}
```
Comment (MrPetovan) [inline src/Module/PermissionTooltip.php]:
```suggestion
			}
```

File: src/Protocol/ActivityPub/ClientToServer.php [hunk 2]
```diff
@@ -222,7 +222,10 @@ private static function getTargets(array $object, string $actor): array
 				case 'as:bcc':
 					$type = Receiver::TARGET_BCC;
 					break;
-			}
+				case 'as:audience':
+					$type = Receiver::TARGET_AUDIENCE;
+					break;
+				}
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub/ClientToServer.php]:
```suggestion
			}
```

File: src/Protocol/ActivityPub/Receiver.php [hunk 3]
```diff
@@ -1165,7 +1166,10 @@ private static function getReceivers(array $activity, string $actor, array $tags
 						case 'as:bcc':
 							$type = self::TARGET_BCC;
 							break;
-					}
+						case 'as:audience':
+							$type = self::TARGET_AUDIENCE;
+							break;
+						}
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub/Receiver.php]:
```suggestion
					}
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
I've changed the merge base, you may have to rebase the branch on develop once it's brought to the new stable release.

**Comment by annando** [comment]
@MrPetovan Conflicts are solved, the PR is now mergeable.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**